scripts/notebook-normalizer/qiskit_docs_notebook_normalizer/__init__.py

```diff
@@ -60,7 +60,6 @@ def main():
         if is_hidden(nb_path):
             continue
 
-        print(f"{nb_path}")
```

Comment:
The print on 78 doesn't make sense anymore. It needs to put the notebook name and probably don't indent things. Maybe also use a ✍️ emoji to make it fun hah